sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/python/GpuPythonHelper.scala

```diff
@@ -100,11 +100,19 @@ object GpuPythonHelper extends Logging {
     })
   }
 
+  // Not sure if need separate worker for databricks, will check it later.
+  private val mapDefaultPythonModules = Map(
+    ("spark", ("rapids.daemon", "rapids.worker")),
+    ("databricks", ("rapids.daemon_databricks", "rapids.worker"))
+  )
+
   // Check the related conf(s) to launch our rapids daemon or worker for
   // the GPU initialization when python on gpu enabled.
   // - python worker module if useDaemon is false, otherwise
   // - python daemon module.
-  private[sql] def checkPythonConfigs(conf: SparkConf): Unit = synchronized {
+  private[sql] def checkPythonConfigs(conf: SparkConf, name: String): Unit = synchronized {
+    val defaultDaemonWorker = mapDefaultPythonModules(name)
```

Comment:
nit:
```
val (daemonModule, workerModule) =  mapDefaultPythonModules(name)
```

Reply:
Will update this in the following PR to avoid approving it again.